client/lib/signup/step-actions.js

```diff
@@ -654,6 +654,23 @@ export function isSiteTopicFulfilled( stepName, defaultDependencies, nextProps )
 	}
 }
 
+/**
+ * Creates a user account and logs them in with an email only.
+ * Returns the dependencies for the step.
+ *
+ * @param {function} callback The username to get suggestions for.
+ * @param {object}   data     POST data object
+ */
+export function onboardPasswordlessUser( callback, { email } ) {
```

Comment:
Similarly to `Undocumented.prototype.usersEmailOnboard`, this symbol name does not indicate to me that an account will be created by virtue of calling this function.  I suggested something like `createUserAccountFromEmailAddress` for that one.  Perhaps this can be something similar.

Reply:
It's not immediately obvious how this function is different from `createPasswordlessUser` -- perhaps a comment to that effect would make it so :)